Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/ServerSettingsSmtpAdminController.cs

```diff
@@ -197,5 +257,26 @@ private static string GetSmtpPassword(int portalId, bool obfuscate)
 
             return pwd;
         }
+
+        private void EnsureUsingMailkitProvider()

```

Comment:
I don't think this is a good idea, as this touches the web.config, updates settings and will trigger an app recycle while the user is interacting with something that they might not expect to have this sort of impact.

Reply:
When user update this setting, it should means they need the setting need to be valid after update, if we do not update the provider, they also need to update it manually, and this recycle will be a one-time action, it only recycle the app when the provider need to be updated.